dapr/clients/__init__.py

```diff
@@ -115,3 +115,47 @@ def invoke_method(
                 metadata=metadata,
                 http_verb=http_verb,
                 http_querystring=http_querystring)
+
+
+    async def invoke_method_async(
+            self,
+            app_id: str,
+            method_name: str,
+            data: Union[bytes, str, GrpcMessage],
+            content_type: Optional[str] = None,
+            metadata: Optional[MetadataTuple] = None,
+            http_verb: Optional[str] = None,
+            http_querystring: Optional[MetadataTuple] = None) -> Awaitable[InvokeMethodResponse]:
+        """Invoke a service method over gRPC or HTTP.
+
+        Args:
+            app_id (str): Application Id.
+            method_name (str): Method to be invoked.
+            data (bytes or str or GrpcMessage, optional): Data for requet's body.
+            content_type (str, optional): Content type of the data.
+            metadata (MetadataTuple, optional): Additional metadata or headers.
+            http_verb (str, optional): HTTP verb for the request.
+            http_querystring (MetadataTuple, optional): Query parameters.
+
+        Returns:
+            Awaitable[InvokeMethodResponse]: the response from the method invocation.
+        """
+        if self.invocation_client:
+            return await self.invocation_client.invoke_method(
+                app_id,
+                method_name,
+                data,
+                content_type=content_type,
+                metadata=metadata,
+                http_verb=http_verb,
+                http_querystring=http_querystring,
+                return_coroutine=True)
+        else:
+            return super().invoke_method(
```

Comment:
Should this else case also be async by adding await?

e.g.
`return await super().invoke_method(

Reply:
```suggestion
            return await super().invoke_method(
```

Reply:
Firstly, thanks for the review and to answer your question/remark; no, I don't think so.
`super()` refers to `DaprGrpcClient`, the `invoke_method` there is completely synchronous. So as far my python knowledge goes, `await` it not needed here.